### ISSUE TITLE
Is it possible to add support for Animaze?

### ISSUE THREAD

**Issue by XieLongWu** [benaclejames/VRCFaceTracking#188]
[Animaze](https://store.steampowered.com/app/1364390?snr=5000_5100___primarylinks) is a desktop face tracking software that supports VRM and Live2D models, and recently I discovered that it also has an [Animaze API](https://[animaze.tv](https://animaze.tv/animaze-api)/animaze-api) that can send data to other addresses.

Is it possible to make a module that supports converting Animaze API data to VRCFT？

**Comment by dfgHiatus** [comment]
Yes, you can implement a module using the VRCFaceTracking.SDK . Feel free to let me know if you have any questions!
